catalog-item.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Каталог товаров</title>
+  </head>
+  <body>
+    <header class="catalog">
+      <div class="container">
+        <div class="logo-header">
+          <a href="#"><img src="img/logo.png" alt=""></a>
+        </div>
+        <nav class="main-navigation">
+          <ul>
+            <li><a href="#">ИНФОРМАЦИЯ</a></li>
+            <li><a href="#">НОВОСТИ</a></li>
+            <li><a href="#">ПРАЙС-ЛИСТ</a></li>
+            <li><a href="#">МАГАЗИН</a></li>
+            <li><a href="#">КОНТАКТЫ</a></li>
+          </ul>
+        </nav>
+        <div class="user-block">
+          <a href="#" class="login">ВХОД</a>
+        </div>
+      </div>
+    </header>
+    <main class="catalog-item">
+      <h1>НАБОР ДЛЯ ПУТЕШЕСТВИЙ «BAXTER OF CALIFORNIA»</h1>
+      <div class="breadcrumbs">
+        <a href="#">ГЛАВНАЯ</a>
+        <a href="#">МАГАЗИН</a>
+        <a href="#">СРЕДСТВА ДЛЯ УХОДА</a>
+        <span>НАБОР ДЛЯ ПУТЕШЕСТВИЙ «BAXTER OF CALIFORNIA»</span>
+      </div>
+      <section class="product-photos">
+        <div class="product-photo-large">
+          <img src="img/baxter-03" alt="">
+        </div>
+        <div class="product-photo-preview">
+          <img src="img/baxter-01" alt="">
+        </div>
+        <div class="product-photo-preview">
+          <img src="img/baxter-02" alt="">
+        </div>
+        <div class="product-photo-preview">
+          <img src="img/baxter-03" alt="">
+        </div>
+      </section>
+      <section class="product-description">
+        <span class="availability">ЕСТЬ В НАЛИЧИИ</span>
+        <span class="product-id">АРТИКУЛ:DEXTER-AE</span>
+        <p>Travel Kit – необходимый аксессуар во время любого путешествия. В аккуратной кожаной сумке находится все, что нужно для бритья и ухода за кожей во время рабочей поездки или отдыха: средство для умывания, увлажняющий крем, крем для бритья, крем после бритья, шампунь. Набор также может стать отличным подарком.</p>
+        <div class="price">2900 РУБ.</div>
+        <button class="btn" name="buy-btn">КУПИТЬ</button>
```

Comment:
It's a good practice to always specify the type of a button. Some browsers treat button as "submit" by default. If this is not what you mean, add type="button" attribute.